mlrun/feature_store/steps.py

```diff
@@ -181,6 +184,10 @@ def do(self, event):
             encoded_values.update(self._encode(feature, val))
         return encoded_values
 
+    def _prepare_category(self, f_name: str):
```

Comment:
Can be static?

Reply:
Probably no need for a method... Just assign this under line 168, e.g.:
```
sanitized_category = re.sub("[ -]", "_", f_name)
```
and use that.